util/naturalsort_test.go

```diff
@@ -0,0 +1,85 @@
+package util
+
+import (
+	"fmt"
+	"math/rand"
+	"sort"
+	"testing"
+)
+
+func shuffleSeries(slice []string) {
+	for i := len(slice) - 1; i > 0; i-- {
```

Comment:
why count down instead of the more commonly used count up?

Reply:
This is the most common version of the Fisher-Yates algorithm. The alternative is from 0 to len-2. I don't really care which one to use.

Reply:
ok then, sounds good